src/styles/blocks/header.scss

```diff
@@ -0,0 +1,59 @@
+.logo {
```

Comment:
Check the 8th point from the [checklist](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md)

Reply:
thanks!

Reply:
But you have it as a block, make that an element of your header block